pyproject.toml

```
@@ -46,7 +46,7 @@ jupyter = [
   'jupyter-server-proxy',
   'nest-asyncio',
   'trame-client>=2.12.7',
-  'trame-server>=2.11.7',
+  'trame-server >= 2.11.7, != 3.7.*, != 3.8.0, ~= 3.8.1',
```

This is easier to read IMO
```suggestion
  'trame-server >= 2.11.7, != 3.7.*, != 3.8.0, < 3.9.0',
```

But in general I don't think we should place an upper bound on user-facing dependencies (we do have an exception for `vtk` though...), so I think it's better to do this:
```suggestion
  'trame-server >= 2.11.7, != 3.7.*, != 3.8.0,
```